spec/models/form/sales/pages/la_nominations_spec.rb

```diff
@@ -3,11 +3,13 @@
 RSpec.describe Form::Sales::Pages::LaNominations, type: :model do
   subject(:page) { described_class.new(page_id, page_definition, subsection) }
 
-  let(:log) { create(:sales_log, :completed) }
+  let(:log) { build(:sales_log, :completed) }
 
   let(:page_id) { nil }
   let(:page_definition) { nil }
-  let(:subsection) { instance_double(Form::Subsection, form: instance_double(Form, start_date: Time.zone.local(2023, 4, 1))) }
+  let(:start_year_after_2024) { false }
+  let(:form) { instance_double(Form, start_date: Time.zone.local(2023, 4, 1), start_year_after_2024?: start_year_after_2024) }
```

Comment:
if we're stubbing the response to :start_year_after_2024?` do we still need the start_date?

Reply:
Yeah, I think it's used in question itself for question number

Reply:
aha.